code/modules/food_and_drinks/drinks/bottler/bottler.dm

```diff
@@ -23,6 +23,7 @@
 	var/bottling = 0
```

Comment:
Boolean here. 0/1 should be FALSE/TRUE

Reply:
Not happening, the whole point of this PR is just to add parent calls, not refactor half the files.